app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>FarMarRails</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+<header class="homeheader">
+  <h1 class = "header">FarMar</h1>
+  <nav>
+    <ul>
+      <li>
+        <%= link_to "Home", root_path%>
+      </li>
+      <li><%= link_to "Markets", markets_path%></li>
+      <li>
+        <%= link_to "Vendors", vendors_path%>
+      </li>
+    </ul>
```

Comment:
Ensure consistent formatting of your lists.  If you are going to add a newline in between the <li> tags as you do for "Home" and "Vendors", you should be consistent across all the list items ("Markets" does not have a line break)